src/import_svg.cc

```diff
@@ -5,72 +5,134 @@
 #include "clipper-utils.h"
 #include "AST.h"
 
+namespace {
+
+constexpr double INCH_TO_MM = 25.4;
+
+struct dim_t {
+	double x;
+	double y;
```

Comment:
Using `Eigen::Vector2d` for this could be interesting since this file isn't part of the svg library. -> future refactoring